### PR TITLE
Document JSON-RPC usage privacy site

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**
I thought it already was on the site, but it's a small addition.
**Context: Fixes an issue? Related issues**
Related to: #717 

It might also go to the release branch

**Status of this Pull Request**
Ready for review

**What is missing until this pull request can be merged?**
Nothing

**Does this need translation?**

YES

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: wiki/en/Privacy-Statement.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. We
 
 ### Use of Profile Information
 
-When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information and the Server operator has no access to it unless they are also connected as using a Client.
+When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information and the Server operator has no access to it unless they are also connected as using a Client or using JSON-RPC.
```
Comment (pljones) [inline wiki/en/Privacy-Statement.md]: This needs clarifying.  Does it mean using JSON-RPC to contact a client or to contact the server?  (Edit "JSON-RPC" just being the transport/protocol, not the actual connection.)

Comment (ann0see) [reply wiki/en/Privacy-Statement.md]: Using JSON-RPC on the server to get a client list?

Comment (pljones) [reply wiki/en/Privacy-Statement.md]: You mean `jamulusserver/getClients`?
image
That's what a Server operator can see when using the Server GUI.
```
"result": {
  "clients": [
    {
      "id": <channel id>,
      "address": <client ip:port>,
      "name": <name>,
      "jitterBufferSize": <jitter buffer size>,
      "channels": <channels>
    }, ...
  }
}
```
That's what `jamulusserver/getClients` returns - i.e. the same thing.  If that _is_ what you mean, it should mention _both_.

`jamulusserver/getServerProfile` gets the Server registration details, nothing about clients (i.e. it's what the client sees in the connect window about this server).

When you are running a Client that's connected to a Server, you can control the Client using JSON-RPC and - just as with the Client GUI, you can receive details of other clients on the server.  `jamulusclient/clientListReceived`  is the same as the Connect window.  If this is what you're referring to, it might be good to give some more context.

In both cases, you need to be running Jamulus as either a Server or a Client and JSON-RPC doesn't expose _any_ information that isn't already exposed by that Server or Client _on the machine that Server or Client is running on and nowhere else_.

Basically the Client and the Server can be "viewed" in two ways: via the GUI or via JSON-RPC.  There's no additional information available over the JSON-RPC connection to the Client or Server than via the GUI (the reverse, in fact, as the Client GUI holds state not known by the Client).  In any case, the information exposed by the GUI and by JSON-RPC is only available on the same machine that the GUI (Client with GUI enabled or Server with GUI enabled) or JSON-RPC (Client with JSON-RPC enabled or Server with JSON-RPC enabled) is being used.

(The same applies with any changes made: if you change something through the GUI or through JSON-RPC, the visibility of that change is the same.)

Comment (ann0see) [reply wiki/en/Privacy-Statement.md]:
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information. The server operator might however use the Server-GUI or the JSON-RPC API to retrieve Profile information (including your IP-Address) while you are connected to a server.
```

Comment (pljones) [reply wiki/en/Privacy-Statement.md]: Better, though it's only the name field of the Client profile.

Comment (ann0see) [reply wiki/en/Privacy-Statement.md]: Isn't the IP included too (at least for JSON-RPC)?

Comment (pljones) [reply wiki/en/Privacy-Statement.md]: The IP/port isn't present in the Client's Profile information.  It's derived from the UDP packet details, not the protocol.  Sending a UDP request to _any_ site and you reveal the source IP and port number.

Comment (ann0see) [reply wiki/en/Privacy-Statement.md]:
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information. The server operator might however use the Server-GUI or the JSON-RPC API to see your Profile information while you are connected to a server.
```

Comment (pljones) [reply wiki/en/Privacy-Statement.md]: It's not the Client Profile information - except for the name.  I think it's misleading to say otherwise.  We don't show the server operator the skill level, instrument, location, language, etc.

Comment (ann0see) [reply wiki/en/Privacy-Statement.md]: Ok. I agree that it can be misleading (it might change in future). Can you give an exact suggestion?

Comment (pljones) [reply wiki/en/Privacy-Statement.md]: Currently, an accurate statement would be 
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it.  Note that, depending on the other client, you may not see them.  Also, the Server operator can see the name you set in My Profile and your IP address, using the Server either through the GUI or JSON-RPC.
```

**Comment by pljones** [comment]
Have we made the following clear regarding the statement?
* It applies only to Jamulus as of the version number with which it's released
* It does not apply to any instance of Jamulus other than one built from unchanged source of the version number with which it's released

File: wiki/en/Privacy-Statement.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Privacy-Statement.md]: > Note that, depending on the other client, you may not see them.

I think what them is referring to is not clear enough. Everything else is OK, I think.

**Comment by ann0see** [comment]
> Have we made the following clear regarding the statement?

It’s not written anywhere on that site, I think.

**Comment by pljones** [comment]
> It’s not written anywhere on that site, I think.

It might be good to state that in the preamble to the statement.

**Comment by ann0see** [comment]
Just a small update on my PRs: I'll look at it in one week since I'll have more time then.

File: wiki/en/Privacy-Statement.md (hunk 1 above)
Comment (ann0see) [reply wiki/en/Privacy-Statement.md]: @pljones could you please clarify what them is referring to: You might not see the other client at all or their information?

**Review by pljones — APPROVED** [review]
